utils/scripts/generate-and-sign-attestations

```diff
@@ -91,7 +91,7 @@ for WHEEL in *.whl *.tar.gz; do
 }
 EOF
 
-  ATT_FILE="${WHEEL}.att"
+  ATT_FILE="${WHEEL}.attestation"
```

Comment:
https://www.mankier.com/1/twine states that twine requires .attestation, not .att
```
Parameters
upload_settings -- The configured options related to uploading to a repository.
dists -- The distribution files to upload to the repository. This can also include .asc and .attestation files, which will be added to their respective file uploads.
```